awswrangler/s3/_read_text.py

```diff
@@ -16,11 +16,11 @@
 from awswrangler.s3._read import _apply_partition_filter, _get_path_ignore_suffix, _get_path_root, _union
 from awswrangler.s3._read_text_core import _read_text_file, _read_text_files_chunked
 
-if config.distributed:
```

Comment:
What happens if the customer happens to have modin on dask installed in their environment? This checks that they have `modin`, and then assumes that they have `dask`, right?

Reply:
Yes it would fail, but our current implementation is not supposed to support dask regardless, only modin on ray. For me the goal of this PR is not to already support other engines, but just to make space for refactoring in the future. To be honest I find this conditionals solution pretty ugly and not scalable. But it's ok for now because it's hidden from the user and can be changed at any point later when we support other engines

Reply:
My point is just that if they have only `modin`, they will pass this `if` statement and get an error telling them an import of some resource in `ray` failed.

The behavior we had so far was to check for both `modin` and `ray`. Shouldn't we keep that behavior now as well?